pkg/util/updater/updater.go

```diff
@@ -121,7 +116,28 @@ func (u *Updater) Update() (err error) {
 		}
 	}()
 
-	return nil
+	return true, nil
+}
+
+// UpdateAvailable checks if an update is available.
+// If it is, the method returns the last available version.
+// Otherwise, it returns nil.
+func (u *Updater) UpdateAvailable() (*Version, error) {
+	u.log.Infof("Looking for updates")
+
+	lastVersion, err := lastVersion()
```

Comment:
Is this meant to be `lastVersion` or `latestVersion`?

Reply:
@evanlinjin `latestVersion` (the newest)